.builds/openbsd.yml

```diff
@@ -22,6 +22,9 @@ packages:
   - libgcrypt
   - libsignal-protocol-c
   - sqlite3
+  - wget
+  - nohup
```

Comment:
Where is this file even used?

If you change it, make it used in CI, otherwise simply remove it IMO